common/cassandra/cassandraCluster.go

```diff
@@ -64,9 +64,6 @@ func NewCassandraCluster(cfg config.Cassandra) *gocql.ClusterConfig {
 	if cfg.MaxConns > 0 {
 		cluster.NumConns = cfg.MaxConns
 	}
-
-	cluster.PoolConfig.HostSelectionPolicy = gocql.TokenAwareHostPolicy(gocql.RoundRobinHostPolicy())
```

Comment:
Is this used in other places besides CLI: Yes
Reason for delete: https://gitmemory.com/issue/gocql/gocql/1274/471217774
Confirmed locally that deleting fixes issue.
Checked with gocql team to confirm that its safe to delete this and yes said yes. Without setting it will will use round robin anyways.

Reply:
Added it back and then unset it only in CLI